frontend/src/components/profile/sidebar/sidebar.css

```diff
@@ -0,0 +1,29 @@
+.sidebar {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 250px;
+    height: 100%;
+    background-color: #f8f9fa;
+    padding: 20px;
+    box-shadow: 2px 0 5px rgba(0, 0, 0, 0.1);
```

Comment:
If position is fixed and height is 100%, I'd recommend we include a vertical overflow of scroll so that when we have more buttons on the sidebar, they'd all be visible